src/main/java/edu/iu/terracotta/repository/QuestionRepository.java

```diff
@@ -2,11 +2,12 @@
 
 import edu.iu.terracotta.model.app.Question;
 import org.springframework.data.jpa.repository.JpaRepository;
-import org.springframework.transaction.annotation.Transactional;
 
 import java.util.List;
 import java.util.Optional;
 
+import javax.transaction.Transactional;
```

Comment:
I forgot to mention this one, I'll fix it

Reply:
Doh! Thanks! :)

Reply:
I blame VS Code.... I just switched a few weeks ago from Eclipse and I'm getting used to the order of syntax suggestions it provides... Eclipse always did the Spring deps 1st in Spring Boot apps